public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -81,6 +82,8 @@ const normalizedPanel = (panel: CustomPanelType): CustomPanelType => ({
 
 export const selectPanelList = (rootState): CustomPanelType[] => rootState.customPanel.panelList;
 
+const {setToast} = useToast();
```

Comment:
For some reason I'm very surprised this works (?)

I think b/c in a previous incantation of `useToast` it had react state (used `setState`) and therefore was required to be inside a React-FC.  (the inception of the `use...` naming).

Glad it works outside too!